src/core/apim.tf

```diff
@@ -207,7 +207,7 @@ resource "azurerm_api_management_api_version_set" "apim_uservice_party_managemen
 
 module "apim_uservice_party_management_v1" {
   source              = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v2.12.5"
-  name                = format("%s-party-mgmt-api-v1", local.project)
+  name                = format("%s-party-mgmt-api", local.project)
```

Comment:
the `-v1` suffix was removed because it will be added automatically (otherwise it will end with `-v1-v1` suffix)